python/tskit/drawing.py

```diff
@@ -37,7 +37,7 @@
 
 def draw_tree(
         tree, width=None, height=None, node_labels=None, node_colours=None,
-        mutation_labels=None, mutation_colours=None, format=None):
+        edge_colours=None, mutation_labels=None, mutation_colours=None, format=None):
```

Comment:
New parameters should go at the end in case any users are using these as position parameters.

Reply:
Good point